app/templates/serviceLearning/slcManagement.html

```diff
@@ -32,7 +32,7 @@ <h1 class="text-center mb-2">{{user.firstName}} {{user.lastName}}'s SLC Proposal
           <td>{{courseDict[course]['creator']}}</td>
           <td>{{courseDict[course]['faculty']|join(", ")}}</td>
           <td>{{courseDict[course]['term'].description}}</td>
-          <td>{{courseDict[course]['status']}}</td>
+          <td id="courseStatus-{{courseDict[course]['id']}}">{{courseDict[course]['status']}}</td>
```

Comment:
Not sure what function the `courseStatus-` serves.

Reply:
I think it would be to make the id name more descriptive. With the without `courseStatus-` then the id will just be an int, this way we know what that int is for without having to trace it back through the code.